packages/teams-js/tsconfig.json

```diff
@@ -3,8 +3,7 @@
   "compilerOptions": {
     "outDir": "./dist",
     "declarationDir": "./dts",
-    "target": "es5",
-    "lib": ["ES5", "DOM", "ES2015.promise"],
+    "lib": ["DOM", "ES2015", "ES5"],
```

Comment:
Keeping ES5 and DOM, changing ES2015 to be all of ES2015 and not just the promise portion.